smarts/core/sensors.py

```diff
@@ -1013,3 +1033,48 @@ def __call__(self, linear_velocity, angular_velocity):
 
     def teardown(self):
         pass
+
+
+class ViaEdgeSensor(Sensor):
```

Comment:
We have been using `via point`, `via lane`, and `via edge`. We should unify them. Both of them are prone to mislead. What we are providing is to allow users to specify a lane + offset position.

Reply:
https://github.com/huawei-noah/SMARTS/pull/212#discussion_r529114482

I do not quite understand, could you clarify. I started with a lane level point, did I unnecessarily change it to edges?

Reply:
Nice, `Via` is more concise